src/logger.rs

```diff
@@ -1,22 +1,33 @@
-use log::{Metadata, Record};
-use crate::logging::DefaultLogger;
 use crate::Logger;
+use crate::Result;
+use log::{Metadata, Record};
+use wapc_guest::host_call;
+use wascc_codec::logging::*;
+use wascc_codec::serialize;
 
-pub struct AutomaticLogger {
-    pub l: DefaultLogger
-}
+/// The reserved capability ID for the logging functionality
+pub const CAPID_LOGGING: &str = "wascc:logging";
+
+const ERROR: usize = 1;
+const WARN: usize = 2;
+const INFO: usize = 3;
+const DEBUG: usize = 4;
+const TRACE: usize = 5;
+
+pub struct AutomaticLogger {}
 
 impl AutomaticLogger {
     pub fn new() -> Self {
         Self::default()
     }
+    fn _log(&self, req: WriteLogRequest) {
```

Comment:
Kind of surprised that a linter let you get away with an underscore-prefixed function name. That's not idiomatic at all.

Reply:
what would be better?  log_impl?